src/shared/helpers/getCourseDate.ts

```diff
@@ -0,0 +1,8 @@
+import { dayJS } from '@/app/services/dayjs';
+
+export const getCourseDate = (startDate: string, staleAfterDays: number) => {
```

Comment:
If we want uniformity, maybe we should not pass a COURSE_STALE_AFTER_DAYS as a prop, but use it directly in this function? 🤔 This will reduce the universality of the function, but guarantees the same behavior across the entire site